NickvisionCavalier.Shared/Controllers/PreferencesViewController.cs

```diff
@@ -241,6 +261,26 @@ public uint AreaMargin
         set => Configuration.Current.AreaMargin = value;
     }
 
+    /// <summary>
+    /// Drawing area X offset (-0.5-0.5)
+    /// </summary>
+    public float AreaOffsetX
```

Comment:
Properties (these with defined get and set) should be inbetween the constructor and first class method